man/sd_notify.xml

```diff
@@ -25,6 +25,9 @@
     <refname>sd_pid_notifyf_with_fds</refname>
     <refname>sd_notify_barrier</refname>
     <refname>sd_pid_notify_barrier</refname>
+    <refname>sd_notify_inline</refname>
+    <refname>sd_notifyf_inline</refname>
```

Comment:
uh, let's please *not* do that.

Why even? THis should not be inline code.  Who is helped by this?